helpers/Physics.h

```diff
@@ -0,0 +1,380 @@
+#ifndef _HELPERS_PHYSICS_H_
+#define _HELPERS_PHYSICS_H_
+
+#include "../include/z64.h"
+#include "Matrix.h"
+
+typedef struct {
+	s32   J; // Amount of strand limbs + 1
```

Comment:
why not change `J` to `numLimbs` and use `numLimbs+1` in code instead of `J`, would be easier to follow